modules/web/web-vhost/src/test/java/com/enonic/xp/web/vhost/impl/VirtualHostInternalHelperTest.java

```diff
@@ -13,6 +13,19 @@
 public class VirtualHostInternalHelperTest
 {
 
+    @Test
+    public void testNormalizePath()
+    {
+        final HttpServletRequest req = mock( HttpServletRequest.class );
+        when( req.getRequestURI() ).thenReturn( "/a/other/service" );
+
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( null ) );
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( "" ) );
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( "/" ) );
+        assertEquals( "/a", VirtualHostInternalHelper.normalizePath( "/ a /" ) );
+        assertEquals( "/a", VirtualHostInternalHelper.normalizePath( "/ a //" ) );
```

Comment:
Don't mix two asserts. Test that `//` is normalized. Maybe the idea is actually to subtract trailing `/` and the rest of normalization is not needed.
Also, find out, maybe sometimes it is better to preserve trailing `/` ?